packages/react/examples/shadow-dom/CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.1.1](http://dequelabs/axe-core-npm/compare/v4.0.0...v4.1.1) (2021-01-28)
```

Comment:
This example is not meant to be a "released" package. We might have a Lerna configuration issue.